chore: add global locale toggle

SalahAdDin opened the pull request:
<!--
Hello 👋 Thank you for submitting a pull request.

To help us merge your PR, make sure to follow the instructions below:

- Create or update the tests
- Create or update the documentation at https://github.com/strapi/documentation
- Refer to the issue you are closing in the PR description: Fix #issue
- Specify if the PR is ready to be merged or work in progress (by opening a draft PR)

Please ensure you read the Contributing Guide: https://github.com/strapi/strapi/blob/main/CONTRIBUTING.md
-->

### What does it do?

It adds the missing `global.localeToggle.label` translation string.

### Why is it needed?

We are getting the following console error in browser:
```
Error: [@formatjs/intl Error MISSING_TRANSLATION] Missing message: "global.localeToggle.label" for locale "es", using default message (Select interface language) as fallback.
    IntlError2 error.js:19
    MissingTranslationError2 error.js:79
    formatMessage message.js:65
    formatMessage2 createIntl.js:29
    LocaleToggle UnauthenticatedLayout.tsx:33
    React 3
 Component Stack: 
    LocaleToggle UnauthenticatedLayout.tsx:27
    div unknown:0
    O2 StyledComponent.ts:255
    Box Box.tsx:144
    header unknown:0
    O2 StyledComponent.ts:255
    Box Box.tsx:144
    O2 StyledComponent.ts:255
    Flex Flex.tsx:26
    div unknown:0
    UnauthenticatedLayout UnauthenticatedLayout.tsx:73
    UseCasePage UseCasePage.tsx:71
    PrivateRoute PrivateRoute.tsx:11
    PrivateUseCasePage unknown:0
    RenderedRoute hooks.tsx:658
    Outlet components.tsx:333
    Suspense unknown:0
    Provider4 createContext.tsx:10
    ConfigurationProvider Configuration.tsx:66
    Provider index.ts:58
    Provider2 Context.tsx:18
    GuidedTourProvider Provider.tsx:57
    TrackingProvider Tracking.tsx:40
    NotificationsProvider Notifications.tsx:52
    Provider4 createContext.tsx:58
    $a093c7e1ec25a057$export$f78649fb9ca566b8 Tooltip.tsx:74
    ot ThemeProvider.tsx:90
    Provider22 context.tsx:10
    DesignSystemProvider DesignSystemProvider.tsx:38
    Theme Theme.tsx:19
    IntlProvider3 provider.js:31
    LanguageProvider LanguageProvider.tsx:17
    Provider index.ts:58
    Provider2 Context.tsx:18
    HistoryProvider BackButton.tsx:71
    Provider index.ts:58
    Provider2 Context.tsx:18
    AuthProvider Auth.tsx:77
    QueryClientProvider2 QueryClientProvider.js:32
    Provider3 Redux
    Provider index.ts:58
    Provider2 Context.tsx:18
    Providers Providers.tsx:34
    App App.tsx:22
    RenderedRoute hooks.tsx:658
    RenderErrorBoundary hooks.tsx:584
    DataRoutes index.tsx:767
    Router components.tsx:420
    RouterProvider index.tsx:490
[<anonymous code>:1:145535](chrome://devtools/content/webconsole/%3Canonymous%20code%3E)
    overrideMethod <anonymous code>:1
    defaultErrorHandler utils.js:20
    formatMessage message.js:65
    formatMessage2 createIntl.js:29
    LocaleToggle UnauthenticatedLayout.tsx:33
    React 14
    notify Redux
    batchedUpdates$1 React
    Redux 4
    s usecase:3
    middleware Redux
    extendCTBInitialDataMiddleware extendCTBInitialData.ts:33
    extendCTBAttributeInitialDataMiddleware extendCTBAttributeInitialData.ts:78
    middleware Redux
    rtkQueryUnauthorizedMiddleware configure.ts:99
    Redux 2
    createImmutableStateInvariantMiddleware Immutable
    createActionCreatorInvariantMiddleware Redux
    dispatch usecase:6
    onChange UnauthenticatedLayout.tsx:39
    handleValueChange SingleSelect.tsx:81
    onChange Select.tsx:162
    25bec8c6f54ee79a useCallbackRef.tsx:15
    setValue useControllableState.tsx:27
    handleSelect Select.tsx:1337
    handleEvent primitive.tsx:10
    React 33
    notify Redux
    batchedUpdates$1 React
    Redux 4
    s usecase:3
    middleware Redux
    extendCTBInitialDataMiddleware extendCTBInitialData.ts:33
    extendCTBAttributeInitialDataMiddleware extendCTBAttributeInitialData.ts:78
    middleware Redux
    rtkQueryUnauthorizedMiddleware configure.ts:99
    Redux 2
    createImmutableStateInvariantMiddleware Immutable
    Redux 3
    step chunk-3VFTDEHI.js:64008
    verb chunk-3VFTDEHI.js:63955
    fulfilled chunk-3VFTDEHI.js:64053
    (Async: promise callback)
    step chunk-3VFTDEHI.js:64066
    __async chunk-3VFTDEHI.js:64068
    __async chunk-3VFTDEHI.js:64050
    Redux 2
```

Checking it, we find the literal is [here](https://github.com/strapi/strapi/blob/develop/packages/core/admin/admin/src/layouts/UnauthenticatedLayout.tsx#L34), but it does not exist in the translation files.

### How to test it?

- Init a new Strapi project.
- Create a new admin user.
- Get to the "Say us something" screen.
- Change the language in the toggler.
- Check the browser console.

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request

hanpaine commented:
Thanks for more improvements on the translations @SalahAdDin 🙏 we'll try and take a look soon

SalahAdDin commented:
@hanpaine Any update on this?

Thank you

Adzouz reviewed (dismissed):
Looks good to me, thanks! Just updated the branch to resolve the conflict and updated the spanish translation to fix a mistake. I'll merge it as soon as the pipelines pass 🚀

SalahAdDin commented:
> Looks good to me, thanks! Just updated the branch to resolve the conflict and updated the Spanish translation to fix a mistake. I'll merge it as soon as the pipelines pass 🚀

Man, It was merged, I cannot believe it.
